static/test/javascripts/spec/common/commercial/badges.spec.js

```diff
@@ -34,11 +34,11 @@ define([
                 var header;
                 switch (sponsorship) {
                     case 'sponsoredfeatures':
-                        header = 'Sponsored by:';
+                        header = 'Supported by:';
                         break;
 
                     case 'advertisement-features':
-                        header = 'Brought to you by:';
+                        header = 'Paid for by:';
```

Comment:
Probably less important but the colon should not be used with the 'Paid for by' label